core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -3295,6 +3346,66 @@ components:
           description: |
             The string-encoded decimal representing the amount of change for the fungible resource.
             A decimal is formed of some signed integer `m` of attos (`10^(-18)`) units, where `-2^(256 - 1) <= m < 2^(256 - 1)`.
+############################################################
+# REQUEST: /lts/stream/account-transactions-basic-outcomes #
+############################################################
+    LtsStreamAccountTransactionsBasicOutcomesRequest:
+      description: |
+        A request to retrieve a sublist of committed transactions from the ledger.
+      type: object
+      required:
+        - network
+        - account_address
+        - from_state_version
+        - limit
+      properties:
+        network:
+          $ref: "#/components/schemas/NetworkIdentifier"
+        account_address:
+          type: string
+          description: The Bech32m-encoded human readable version of the account's address
+        from_state_version:
+          type: integer
+          format: int64
+          minimum: 1
+          maximum: 100000000000000
+          description: An integer between `1` and `10^13`, giving the first (resultant) state version to be returned
+        limit:
+          description: The maximum number of transactions that will be returned.
+          type: integer
+    LtsStreamAccountTransactionsBasicOutcomesResponse:
+      type: object
+      required:
+        - from_state_version
```

Comment:
To enable this endpoint to be efficiently polled, we may wish to provide a `next_state_version` which is the state version at which the iteration should start. (ie one more than the max the iterator was at this response).

In particular, even if no entries were returned, it should probably give `max_ledger_state_version + 1` I guess - to save re-scanning over things that have already been checked.

I guess this isn't super important if we have an efficient index and aren't doing any additional filtering beyond using the index; but it's possibly a good pattern regardless?

Happy to hear your thoughts :)

Reply:
Good point if `limit` is referring to the `state_version` and indeed, the last transaction outcome is not the last scanned. I consider `limit` refers to the number of transactions returned, and in this case, the last transaction would basically have this information.

I'll change this if it makes sense in a later PR.